### PR TITLE
Add pgxn config update executon pipeline

### PR DESCRIPTION
**Pull request by gurkanindibay**

### CODE REVIEW

File: .github/workflows/update-pgxn-version.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        run: sudo apt install libcurl4-openssl-dev libssl-dev
+
+      - name: Clone Tools branch
+        run: git clone --branch tools-transition-update-docker https://github.com/citusdata/tools.git
+
+      - name: Install Python requirements
+        run: export PYTHONPATH="$(pwd)/tools" && python -m pip install -r tools/packaging_automation/requirements.txt
+
+      - name: Update Docker Files
+        run: python -m packaging_automation.update_pgxn --prj_ver ${PROJECT_VERSION} --exec_path "$(pwd)" --gh_token ${GH_TOKEN}
```
Comment (hanefi) [inline .github/workflows/update-pgxn-version.yml]: missing empty line at the end

**Review by hanefi — APPROVED** [review]
One minor comment

File: .github/workflows/update-pgxn-version.yml (hunk 1 above)
Comment (hanefi) [reply .github/workflows/update-pgxn-version.yml]:
```suggestion
        run: python -m packaging_automation.update_pgxn --prj_ver ${PROJECT_VERSION} --exec_path "$(pwd)" --gh_token ${GH_TOKEN}
        
```

Comment (gurkanindibay) [reply .github/workflows/update-pgxn-version.yml]: Fixed

**Pull request merged**